root/components/list/ReleaseList.js

```diff
@@ -107,24 +107,21 @@ const ReleaseList = ({
         <th>
           {sortable
             ? (
-              <SortableTableHeader
-                label={l('Date')}
-                name="date"
-                order={order}
-              />
-            )
-            : l('Date')}
-        </th>
-        <th>
-          {sortable
-            ? (
-              <SortableTableHeader
-                label={l('Country')}
-                name="country"
-                order={order}
-              />
+              <>
+                <SortableTableHeader
+                  label={l('Country')}
+                  name="country"
+                  order={order}
+                />
+                {lp('/', 'inclusive or')}
+                <SortableTableHeader
+                  label={l('Date')}
+                  name="date"
+                  order={order}
+                />
+              </>
             )
-            : l('Country')}
+            : l('Country') + lp('/', 'inclusive or') + l('Date')}
```

Comment:
I’m not sure that `inclusive or` is self-explanatory to anyone who isn’t programming.
I would suggest `table-header-column-separator` or something instead...

Reply:
I tried to pick something more general that could be used in other places we use a slash (though the only other place I can think of is language/script pairs). `lp('/', 'and')` seems a lot better than "inclusive or" now that I think about it. Do you think that'd be useful?

Reply:
I feel like it still is missing _context_ but I don’t have any better suggestion.